### PR TITLE
enabled mrview gui

### PR DESCRIPTION
**Pull request by stebo85**
maybe also helpful for #338

### CODE REVIEW

**Comment by kaczmarj** [comment]
thanks @stebo85 - i'll test this out

**Comment by stebo85** [comment]
I recently build a new image and these three lines
      [ ! -f /usr/bin/moc ] && ln -s /usr/bin/moc-qt5 /usr/bin/moc
      [ ! -f /usr/bin/qmake ] && ln -s /usr/bin/qmake-qt5 /usr/bin/qmake
      [ ! -f /usr/bin/rcc ] && ln -s /usr/bin/rcc-qt5 /usr/bin/rcc
where necessary when I made this pull request, but don't seem to be necessary anymore.

**Comment by kaczmarj** [comment]
thanks @stebo85 - i will review this PR (and the other PRs!) soon. thank you so much for your contributions.

**Comment by civier** [comment]
@stebo85, the MRtrix singularity image from CVL works fine with graphics (even with Mesa software rendering, if I enforce OpenGL 3.3 through Env variables). However, I understand this specific CVL singularity image is not built using neurodocker.
I wonder -- are the recipes for the CVL containers (non-neurodocker ones) available online somewhere? If not, do you have access to them and can share?

**Comment by stebo85** [comment]
Dear @civier , the MRtrix singularity image is built using neurodocker: https://github.com/NeuroDesk/caid/blob/master/mrtrix3/build.sh and hosted on dockerhub (https://hub.docker.com/orgs/vnmd/repositories) and my singularity container storage (curl -s -S -X GET https://swift.rc.nectar.org.au:8888/v1/AUTH_d6165cc7b52841659ce8644df1884d5e/singularityImages)

**Comment by satra** [comment]
let's update this once #370 is merged. would be great to get mrview in :)

File: neurodocker/templates/mrtrix3.yaml [hunk 1]
```diff
@@ -34,6 +35,9 @@ generic:
       {% if mrtrix3.version != "master" and mrtrix3.version != "latest" -%}
       git checkout {{ mrtrix3.version }}
       {% endif -%}
-      ./configure -nogui
+      [ ! -f /usr/bin/moc ] && ln -s /usr/bin/moc-qt5 /usr/bin/moc
+      [ ! -f /usr/bin/qmake ] && ln -s /usr/bin/qmake-qt5 /usr/bin/qmake
+      [ ! -f /usr/bin/rcc ] && ln -s /usr/bin/rcc-qt5 /usr/bin/rcc
```
Comment (satra) [inline neurodocker/templates/mrtrix3.yaml:40]:
```suggestion
```

if these are not necessary, can we remove them?

**Comment by stebo85** [comment]
mrview GUI is added in neurodesk build of mrtrix container

**Pull request closed without merging**